docroot/themes/humsci/humsci_basic/Gruntfile.js

```diff
@@ -0,0 +1,84 @@
+const sass = require('node-sass');
+const glob = require('glob');
+
+module.exports = function (grunt) {
+
+  function getIncludeFiles() {
+    const patterns = [
+      '**/*.scss',
+      'node_modules/decanter/scss',
+    ];
+
+    const libraries = [];
+    patterns.map(function (pattern) {
+      glob.sync(pattern).map(function (file) {
+        libraries.push(file);
+      });
+    });
+
+    return libraries;
+  }
+
+  grunt.initConfig({
+    pkg: grunt.file.readJSON('package.json'),
+    watch: {
+      css: {
+        files: ['**/*.{scss,sass}'],
+        tasks: ['sass', 'run:stylelint'], // run stylelint at same time after each change
+        options: {
+          interrupt: true
+        }
+      }
+    },
+    run: {
+      stylelint: {
+        exec: 'npm run lint:css'
+      }
+    },
+    postcss: {
+      options: {
+        map: true, // inline sourcemaps
+        processors: [
+          require('autoprefixer')({
```

Comment:
FYI our browserlist config: https://github.com/SU-SWS/stanford_basic/blob/8.x-4.x/.browserslistrc

Reply:
got it 👍  this is defined in our package.json